floodviz/hydrograph_utils.py

```diff
@@ -38,9 +39,11 @@ def parse_hydrodata(jdata):
             dt_ms = time.mktime(dt.timetuple()) * 1000
             # (for below if statment) create dummy value for nvd3 issue at https://github.com/novus/nvd3/issues/695 #
             if idx2 is 0: # First datapoint of this site
-                all_series_data[idx]['values'].append({'date': date, "time": 0, "time_mili": dt_ms, 'value': 0})
+                all_series_data[idx]['values'].append({'site_name': site_name, 'date': date, "time": 0,
```

Comment:
Should we maybe have sitename as another key on line 28 instead of having it recorded for every datapoint?